snapcraft.yaml

```diff
@@ -27,9 +29,18 @@ parts:
       - lib
       - sbin
     stage-packages:
+      - coreutils
```

Comment:
/bin/dd and /bin/cp are no longer listed in prime, but coreutils is still in stage-packages; this should also be dropped.